scripts/setup.sh

```
@@ -25,173 +25,182 @@ echo ""
-    exit 1
+  echo "Failed to apply CA issuer prerequisites"
+  exit 1
 }
 wait_for_resource clusterissuer/default-ca condition=Ready 300
 
 # Apply authorino prerequisites and wait for it to be ready
 if oc get deployment authorino-operator -n openshift-operators &>/dev/null; then
-    echo "Authorino operator is already installed, skipping..."
+  echo "Authorino operator is already installed, skipping..."
 else
+<<<<<<< Updated upstream
```

Shellcheck (0.11.0)

[error] Couldn't parse this here string. Fix to allow more checks.

(SC1073)